lua/vgit/git/cli/Git.lua

```diff
@@ -12,6 +12,8 @@ local Status = require('vgit.git.cli.models.Status')
 
 local Git = Object:extend()
 
+local log_line_format = '%H<>%P<>%at<>%an<>%ae<>%s'
```

Comment:
Git doesn't have any limitation on what characters can be contained in the author name or the commit message, so any character sequence will theoretically have the same issue as `-`.

Here I just picked an arbitrary separator that I suppose will be much less common.